src/recipe.rs

```diff
@@ -63,6 +63,16 @@ impl<'src, D> Recipe<'src, D> {
     self.name.line
   }
 
+  pub(crate) fn confirm(&self) -> bool {
```

Comment:
This function should be fallible, since `read_line` can fail, so it should return `RunResult<bool>`